src/blueprint/azext_blueprint/_params.py

```diff
@@ -0,0 +1,215 @@
+        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('tags', tags_type, arg_group='Resource Group', help='Tags to be assigned to this resource group.')
+
+    with self.argument_context('blueprint resource-group delete') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='A unique name of this resource group artifact.')
+
+    with self.argument_context('blueprint resource-group show') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='A unique name of this resource group artifact.')
+
+    with self.argument_context('blueprint resource-group list') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+
+    with self.argument_context('blueprint artifact policy create') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='Name of the blueprint artifact.')
+        c.argument('display_name', help='DisplayName of this artifact.')
+        c.argument('description', help='Description of the blueprint artifact.')
+        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('policy_definition_id', help='Azure resource ID of the policy definition.')
+        c.argument('resource_group_art', help='Name of the resource group artifact to which the policy will be assigned.')
+        c.argument('parameters', arg_type=parameter_type)
+
+    with self.argument_context('blueprint artifact policy update') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='Name of the blueprint artifact.')
+        c.argument('display_name', help='DisplayName of this artifact.')
+        c.argument('description', help='Description of the blueprint artifact.')
+        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('resource_group_art', help='Name of the resource group artifact to which the policy will be assigned.')
+        c.argument('parameters', arg_type=parameter_type)
+
+    with self.argument_context('blueprint artifact role create') as c:
+        c.argument('blueprint_name', help='Name of the blueprint definition.')
+        c.argument('artifact_name', help='Name of the blueprint artifact.')
+        c.argument('display_name', help='DisplayName of this artifact.')
+        c.argument('description', help='Description of the blueprint artifact.')
+        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('role_definition_id', help='Azure resource ID of the RoleDefinition.')

```

Comment:
The same as policy definition id.

Reply:
The same as policy since it's an artifact, only artifact_name is needed.